reference/docs-conceptual/learn/shell/creating-profiles.md

```diff
@@ -0,0 +1,153 @@
+---
+description: >
+  This article explains how to use your profile to save preferred PowerShell settings and optimize
+  your shell experience.
+ms.date: 07/28/2022
+title: Customizing your shell environment
+---
+# Customizing your shell environment
+
+A PowerShell profile is a script that runs when PowerShell starts. You can use the profile to
+customize the environment. You can add aliases, functions, variables, load modules, create
+PowerShell drives, and change preference settings. Putting these settings in your profile ensures
+they are available whenever you start PowerShell on your system.
```

Comment:
I don't love this, but I wanted to add `run commands` explicitly and the construction of `add aliases, functions, variables, load modules...` didn't sit quite right.

```suggestion
customize the environment. You can:

- add aliases, functions, and variables
- load modules
- create PowerShell drives
- run arbitrary commands
- and change preference settings.

Putting these settings in your profile ensures they are available whenever you start PowerShell on
your system.
```